### PR TITLE
Add caption kenleung pw

### PR DESCRIPTION
**Pull request by kenleung-PW**
modified `17-kenleung-PW.md`

### CODE REVIEW

File: _slides/17-kenleung-PW.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello everyone!
```
Comment (dev-juchen) [inline _slides/17-kenleung-PW.md:6]: Line comment

**Review by dev-juchen — APPROVED** [review]
General review comment

**Review by ArthurPricePW — APPROVED** [review]
Nice Work

**Comment by mkhanukayev** [comment]
PR is approved

**Comment by mkhanukayev** [comment]
PR is approved final

**Pull request merged**